src/NzbDrone.Common/Disk/DiskTransferService.cs

```diff
@@ -511,6 +502,21 @@ private void TryMoveFileVerified(string sourcePath, string targetPath, long orig
             }
         }
 
+        private void VerifyFile(string sourcePath, string targetPath, long originalSize, string action)
+        {
+            var targetSize = _diskProvider.GetFileSize(targetPath);
+            if (targetSize != originalSize)
+            {
+                _logger.Info("File {0} incomplete, waiting in case filesystem is not synchronized. [{1}] was {2} bytes long instead of the expected {3}.", action, targetPath, targetSize, originalSize));
+                WaitForIO();
+            }
+            targetSize = _diskProvider.GetFileSize(targetPath);
+            if (targetSize != originalSize)
```

Comment:
Similar thing here, with a new line after the if block and an early return if equal

Reply:
👍 Thanks for the advice, updated!